.github/workflows/main.yml

```diff
@@ -101,6 +165,15 @@ jobs:
     if: github.event_name == 'push' && github.repository == 'junit-team/junit5' && (startsWith(github.ref, 'refs/heads/releases/') || github.ref == 'refs/heads/main')
     steps:
     - uses: actions/checkout@v2
+    - name: Cache Gradle wrapper and dependencies
```

Comment:
This step and the one below don't invoke the Maven integration tests, so no need to cache .m2 here.